src/main/java/com/getbase/models/Lead.java

```diff
@@ -62,6 +63,10 @@ public Long getOwnerId() {
     return this.ownerId;
   }
 
+  public Long getSourceId() {
```

Comment:
What about equals and hashcode methods? Don't you need to include source id there as well?